hamilton/driver.py

```diff
@@ -898,6 +900,28 @@ def visualize_execution(
             show_schema=show_schema,
         )
 
+    @capture_function_usage
+    def export_execution(
+        self,
+        final_vars: List[str],
+        inputs: Dict[str, Any] = None,
+        overrides: Dict[str, Any] = None,
+    ) -> str:
+        """Method to create JSON representation of the Graph.
+
+        :param final_vars: The final variables to compute.
+        :param inputs: Optional. The inputs to the DAG.
+        :param overrides: Optional. Overrides to the DAG.
+        :return: JSON string representation of the graph.
+        """
+        nodes, user_nodes = self.graph.get_upstream_nodes(final_vars, inputs, overrides)
+        Driver.validate_inputs(self.graph, self.adapter, user_nodes, inputs, nodes)
+        all_nodes = nodes | user_nodes
+
+        hamilton_nodes = [HamiltonNode.from_node(n).as_dict() for n in all_nodes]
+        sorted_nodes = sorted(hamilton_nodes, key=lambda x: x["name"])
+        return json.dumps(sorted_nodes)
```

Comment:
Thoughts -- should we put this under a key "nodes"? That way it leaves room for other things (not sure what they would be but flexibility is nice).

Reply:
Yes I agree. As this is a public method it will be harder to change in the future so we should make it extensible. I also don't see any downsides to putting them under the key `nodes`. I'll update the PR later today.